landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
+        )
+        response.raise_for_status()
+    except (requests.HTTPError, requests.ConnectionError):
```

Comment:
This is a somewhat usless exception to send to sentry. If the /__lbheartbeat is down for lando-api we will already know that due to cloud ops monitoring. This exception does not provide any useful information as to why landoapi is down.

In my opinion a better solution is to check if the status code is 200 and if not return a general 500 with a simple if statement - no try/except